internal/namespaces/container/v1beta1/containerutils/get_or_create.go

```diff
@@ -0,0 +1,78 @@
+package containerutils
```

Comment:
Why a specific package is required?
utils seems to be ill named https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common 
I would be more of namespace and container-specific packages.